scene/3d/bone_attachment_3d.cpp

```diff
@@ -112,5 +364,21 @@ void BoneAttachment3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_bone_name", "bone_name"), &BoneAttachment3D::set_bone_name);
```

Comment:
Setting the bone name does not set the bone idx in my testing for bone attachments.